src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -473,6 +496,39 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 	return nil
 }
 
+func (c controller) getReadyOperators() ([]string, []models.MonitoredOperator, error) {
+	var readyOperators []string
+	operators, err := c.ic.GetClusterMonitoredOLMOperators(context.TODO(), c.ClusterID)
+	if err != nil {
+		return readyOperators, operators, err
+	}
+
+	for index := range operators {
+		handler := NewClusterServiceVersionHandler(c.kc, &operators[index], c.Status)
```

Comment:
why not to create handlers once?and provide them to function?

Reply:
Done.